apps/ts-minbar-test-react-components/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "ts-minbar-test-react-components",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Testing Fluent UI React Components compatibility with Typescript 3.9",
+  "license": "MIT",
+  "dependencies": {
+    "@fluentui/react-components": "^9.0.0-beta.3"
+  },
+  "scripts": {
+    "build": "just-scripts build",
+    "test": "node -r @fluentui/scripts/babel/register src/index.ts"
```

Comment:
Is there sense you use `@babel/register` there? On CI `lage` will run `build` anyway first.
`@fluentui/scripts/babel/register` is also very specific to Northstar itself 🙄

Reply:
- The babel setup might just be because this test was started by copying from the `cra-template` test (which was copied from `projects-test`)
- I suggested he set up build with `noEmit` for just type checking, though we could also get rid of that and just do `node ./lib/index`
- The "v8-ish" way would be to run the test using `node -r @fluentui/scripts/ts-node-register`